sdk/pre-authorized-debit-v1/src/read/implementation/index.ts

```diff
@@ -65,7 +66,9 @@ export class PreAuthorizedDebitReadClientImpl
   }
 
   public async fetchIdl(): Promise<Idl> {
-    const idl = await Program.fetchIdl(this.programId);
+    const idl = await Program.fetchIdl(this.programId, {
+      connection: this.connection,
+    });
```

Comment:
This was a bug

Reply:
I renamed the function, can you remove this one?

Reply:
already done